src/main/java/net/dv8tion/jda/internal/requests/WebSocketSendingThread.java

```diff
@@ -116,9 +116,9 @@ public void run()
             queueLock.lockInterruptibly();
 
             audioRequest = client.getNextAudioConnectRequest();
-            chunkOrSyncRequest = chunkSyncQueue.peek();
-            if (chunkOrSyncRequest != null)
-                handleChunkSync(chunkOrSyncRequest);
+            chunkRequest = chunkSyncQueue.peek();
```

Comment:
Should we rename `chunkSyncQueue` to `chunkQueue`?

Reply:
Yes, I already did in the WebSocketClient but forgot it here.